### PR TITLE
Heatmap2

### PR DESCRIPTION
**Pull request by lukedirtwalker**
Replaces #164
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/dimitar-asenov/Envision/pull/166%23issuecomment-146467593%22%2C%20%22https%3A//github.com/dimitar-asenov/Envision/pull/166%23issuecomment-146467963%22%2C%20%22https%3A//github.com/dimitar-asenov/Envision/pull/166%23issuecomment-146468361%22%2C%20%22https%3A//github.com/dimitar-asenov/Envision/pull/166%23discussion_r41492558%22%2C%20%22https%3A//github.com/dimitar-asenov/Envision/pull/166%23discussion_r41493886%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/dimitar-asenov/Envision/pull/166%23issuecomment-146467593%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22I%20noticed%20that%20you%20removed%20the%20%60blueHighlight%60%20and%20%60greenHighlight%60%20files.%20Do%20you%20not%20need%20these%20anymore%3F%20They%20were%20there%20before%20this%20feature.%22%2C%20%22created_at%22%3A%20%222015-10-08T09%3A16%3A53Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1152976%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dimitar-asenov%22%7D%7D%2C%20%7B%22body%22%3A%20%22No%20those%20are%20not%20needed%20since%20they%20only%20had%20a%20different%20color%20stored%20in%20them.%20Now%20we%20can%20set%20any%20color.%22%2C%20%22created_at%22%3A%20%222015-10-08T09%3A19%3A19Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3530769%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/lukedirtwalker%22%7D%7D%2C%20%7B%22body%22%3A%20%22But%2C%20in%20this%20PR%2C%20you%20don%27t%20change%20any%20code%20where%20these%20styles%20were%20used%3F%20Is%20that%20code%20broken%20now%2C%20or%20did%20it%20never%20exist%3F%22%2C%20%22created_at%22%3A%20%222015-10-08T09%3A20%3A29Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1152976%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dimitar-asenov%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%206f59a26e328f208d1fcc77b32cadebc48ef23c71%20InformationScripting/src/visualization/DefaultVisualizer.cpp%2014%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/dimitar-asenov/Envision/pull/166%23discussion_r41492558%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%40dimitar-asenov%20The%20styles%20were%20used%20here%20before%2C%20nowhere%20else.%22%2C%20%22created_at%22%3A%20%222015-10-08T09%3A21%3A47Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3530769%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/lukedirtwalker%22%7D%7D%2C%20%7B%22body%22%3A%20%22Ohh%2C%20I%20see.%20I%20searched%20for%20%5C%22geenHighlight%5C%22%20%3A%29%22%2C%20%22created_at%22%3A%20%222015-10-08T09%3A34%3A39Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1152976%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dimitar-asenov%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20InformationScripting/src/visualization/DefaultVisualizer.cpp%3AL111-130%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/dimitar-asenov/Envision/pull/166#issuecomment-146467593'>General Comment</a></b>
- <a href='https://github.com/dimitar-asenov'></a> I noticed that you removed the `blueHighlight` and `greenHighlight` files. Do you not need these anymore? They were there before this feature.
- <a href='https://github.com/lukedirtwalker'></a> No those are not needed since they only had a different color stored in them. Now we can set any color.
- <a href='https://github.com/dimitar-asenov'></a> But, in this PR, you don't change any code where these styles were used? Is that code broken now, or did it never exist?
- [ ] <a href='#crh-comment-Pull 6f59a26e328f208d1fcc77b32cadebc48ef23c71 InformationScripting/src/visualization/DefaultVisualizer.cpp 14'></a> &nbsp;<b><a href='https://github.com/dimitar-asenov/Envision/pull/166#discussion_r41492558'>File: InformationScripting/src/visualization/DefaultVisualizer.cpp:L111-130</a></b>
- <a href='https://github.com/lukedirtwalker'></a> @dimitar-asenov The styles were used here before, nowhere else.
- <a href='https://github.com/dimitar-asenov'></a> Ohh, I see. I searched for "geenHighlight" :)

<a href='https://www.codereviewhub.com/dimitar-asenov/Envision/pull/166?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/dimitar-asenov/Envision/pull/166?approve=1'></a>&nbsp;<a href='https://github.com/dimitar-asenov/Envision/pull/166'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by dimitar-asenov** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/dimitar-asenov/Envision/pull/166'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by dimitar-asenov** [comment]
I noticed that you removed the `blueHighlight` and `greenHighlight` files. Do you not need these anymore? They were there before this feature.

**Comment by lukedirtwalker** [comment]
No those are not needed since they only had a different color stored in them. Now we can set any color.

**Comment by dimitar-asenov** [comment]
But, in this PR, you don't change any code where these styles were used? Is that code broken now, or did it never exist?

File: InformationScripting/src/visualization/DefaultVisualizer.cpp [hunk 1]
```diff
@@ -111,13 +111,19 @@ void DefaultVisualizer::visualize(const TupleSet& ts)
 			while (nodeVisualizationIt != Visualization::Item::nodeItemsMap().end() && nodeVisualizationIt.key() == node)
 			{
 				auto item = *nodeVisualizationIt++;
-				auto overlay = new Visualization::SelectionOverlay(
```
Comment (lukedirtwalker) [inline InformationScripting/src/visualization/DefaultVisualizer.cpp:114]: @dimitar-asenov The styles were used here before, nowhere else.

Comment (dimitar-asenov) [inline InformationScripting/src/visualization/DefaultVisualizer.cpp:114]: Ohh, I see. I searched for "geenHighlight" :)

**Pull request merged**